.settings/org.eclipse.m2e.core.prefs

```diff
@@ -1,4 +0,0 @@
-activeProfiles=
-eclipse.preferences.version=1
-resolveWorkspaceProjects=true
-version=1
```

Comment:
Please make sure in future that you don't push something unnecessary to store on git.